msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -360,6 +361,19 @@ public PublicClientApplicationConfiguration getConfiguration() {
         void onAccountsLoaded(List<IAccount> accounts);
     }
 
+    /**
+     * Listener callback for asynchronous finding of IAccount matching the given oid.
+     */
+    public interface AccountLoadedCallback {
```

Comment:
Can we re-tool the existing interface to accommodate returning 1 _or_ many accounts? I think we can keep the method name plural without confusing ppl

Something like
```java
public interface AccountsLoadedCallback<T> {
    void onAccountsLoaded(T account);
}
```

And then our various implementations will pass-in the right arg depending on what's needed
```java
new AccountsLoadedCallback<Account>() { // Just need one
    @Override
    public void onAccountsLoaded(Account account) {
        // Do something useful            
    }
}

new AccountsLoadedCallback<List<Account>>() { // Need multiple
    @Override
    public void onAccountsLoaded(List<Account> account) {
        // Something else! 
    }
}
```